java/src/main/native/ai_onnxruntime_OrtEnvironment.c

```diff
@@ -12,7 +12,8 @@
  * Method:    createHandle
  * Signature: (Ljava/lang/String;)J
  */
-JNIEXPORT jlong JNICALL Java_ai_onnxruntime_OrtEnvironment_createHandle(JNIEnv * jniEnv, jclass jobj, jlong apiHandle, jint loggingLevel, jstring name) {
+JNIEXPORT jlong JNICALL Java_ai_onnxruntime_OrtEnvironment_createHandle__JILjava_lang_String_2
```

Comment:
>__JILjava_lang_String_2 [](start = 71, length = 23)

Out of scientific curiosity, why the name change?

Reply:
The names are automatically mangled by JNI, as it's emitting a C header and we don't have function overloading. There are two createHandle methods now, one with a threading options, one without. I guess I could pass a 0 in and unify them, but they call different functions in the c api so it wouldn't be much shorter.